great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -145,6 +151,8 @@ def __init__(
         if variables is None:
             variables = {}
 
+        self._usage_statistics_handler = usage_statistics_handler
```

Comment:
@cdkini Non-blocking question -- should we be grabbing the main `DataContext` handler (please see `Checkpoint`).

Reply:
it's being supplied from the context in the other constructor! This is to decouple the two objects so we can eventually remove the context as a dep